numpy-stubs/__init__.pyi

```diff
@@ -803,6 +803,7 @@ _Mode = Literal["raise", "wrap", "clip"]
 _Order = Literal["C", "F", "A"]
 _PartitionKind = Literal["introselect"]
 _SortKind = Literal["quicksort", "mergesort", "heapsort", "stable"]
+_Side = Literal["left", "right"]
```

Comment:
Looks like this isn't being used?

Reply:
Good call; this is still a leftover from https://github.com/numpy/numpy-stubs/pull/71.